handler/actions/src/test/java/io/knotx/fragments/handler/action/http/HttpActionFactoryTest.java

```diff
@@ -20,13 +20,17 @@
 
 import io.knotx.fragments.handler.action.exception.ActionConfigurationException;
 import io.knotx.fragments.handler.api.Cacheable;
+import io.netty.handler.codec.http.HttpMethod;
 import io.vertx.core.Vertx;
 import io.vertx.core.json.JsonObject;
 import io.vertx.junit5.VertxExtension;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.junit.jupiter.params.provider.ValueSource;
 
 @ExtendWith(VertxExtension.class)
 class HttpActionFactoryTest {
```

Comment:
Can we check that all `IllegalArgumentException`s are transformed to `ActionConfigurationException`s?

Reply:
We can also add tests when both `body` and `bodyAsJson` are configured.